airbyte-integrations/connectors/destination-rockset/src/test-integration/java/io/airbyte/integrations/destination/rockset/RocksetDestinationAcceptanceTest.java

```diff
@@ -0,0 +1,192 @@
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.integrations.destination.rockset;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Lists;
+import com.rockset.client.ApiClient;
+import com.rockset.client.api.QueriesApi;
+import com.rockset.client.model.QueryRequest;
+import com.rockset.client.model.QueryRequestSql;
+import com.squareup.okhttp.Response;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.lang.Exceptions;
+import io.airbyte.commons.resources.MoreResources;
+import io.airbyte.integrations.standardtest.destination.DestinationAcceptanceTest;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.stream.Collectors;
+
+import org.junit.jupiter.api.AfterAll;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.collections.Sets;
+
+public class RocksetDestinationAcceptanceTest extends DestinationAcceptanceTest {
+
+    private static final ObjectMapper mapper = new ObjectMapper();
+    private static final Set<String> collectionsToClear = Sets.newHashSet();
+    private static final Set<String> collectionsToDelete = Sets.newHashSet();
+    private static final ExecutorService tearDownExec = Executors.newCachedThreadPool();
+    private static final RocksetSQLNameTransformer nameTransformer = new RocksetSQLNameTransformer();
+
+    private static final Logger LOGGER =
+            LoggerFactory.getLogger(RocksetDestinationAcceptanceTest.class);
+
+    @Override
+    protected String getImageName() {
+        return "airbyte/destination-rockset:dev";
+    }
+
+    @Override
+    protected JsonNode getConfig() throws IOException {
+        return Jsons.deserialize(MoreResources.readResource("secrets/config.json"));
```

Comment:
Most destination connectors we have are using `IOs.readFile(Path.of("secrets/config.json")`. I think this is why this test cannot find the `secrets/config.json` file and when I use `IOs.readFile(Path.of("secrets/config.json")` it fixes this.